src/Database/Expression/WindowExpression.php

```diff
@@ -35,6 +36,16 @@ class WindowExpression implements ExpressionInterface, WindowInterface
      */
     protected $order;
 
+    /**
+     * @var array
+     */
+    protected $frame;
```

Comment:
Wouldn't it be better to intialize the property to empty array?

Reply:
I can if that's cleaner, but it doesn't make a difference in terms of implementation since the array is assigned when configured.